packages/@aws-cdk/pipelines/test/integ.pipeline-with-logging.ts

```diff
@@ -0,0 +1,60 @@
+/// !cdk-integ PipelineStack pragma:set-context:@aws-cdk/core:newStyleStackSynthesis=true
```

Comment:
This doesn't do anything with our new test runner. We need to clean this up to stop confusing people. But also, this test is fantastic. Thank you for the thorough testing!